<?xml version="1.0" encoding="utf-8"?>
<TestRecord xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
  <Results>
    <UnitTestResult>
      <TestDate>2014-09-09T11:17:09</TestDate>
      <Passed>1</Passed>
      <Errors>0</Errors>
      <Failures>0</Failures>
      <Inconclusive>0</Inconclusive>
      <NotRunnable>0</NotRunnable>
      <Skipped>0</Skipped>
      <Ignored>0</Ignored>
      <Time />
      <Message>Test successful

Execution time: 0.06ms</Message>
    </UnitTestResult>
    <UnitTestResult>
      <TestDate>2014-09-09T11:20:56</TestDate>
      <Passed>1</Passed>
      <Errors>0</Errors>
      <Failures>1</Failures>
      <Inconclusive>0</Inconclusive>
      <NotRunnable>0</NotRunnable>
      <Skipped>0</Skipped>
      <Ignored>0</Ignored>
      <Time />
      <Message>One or more child tests had errors</Message>
    </UnitTestResult>
    <UnitTestResult>
      <TestDate>2014-09-09T11:21:12</TestDate>
      <Passed>1</Passed>
      <Errors>0</Errors>
      <Failures>1</Failures>
      <Inconclusive>0</Inconclusive>
      <NotRunnable>0</NotRunnable>
      <Skipped>0</Skipped>
      <Ignored>0</Ignored>
      <Time />
      <Message>One or more child tests had errors</Message>
    </UnitTestResult>
    <UnitTestResult>
      <TestDate>2014-09-09T11:29:07</TestDate>
      <Passed>0</Passed>
      <Errors>0</Errors>
      <Failures>2</Failures>
      <Inconclusive>0</Inconclusive>
      <NotRunnable>0</NotRunnable>
      <Skipped>0</Skipped>
      <Ignored>0</Ignored>
      <Time />
      <Message>One or more child tests had errors</Message>
    </UnitTestResult>
  </Results>
  <Tests>
    <TestRecord Name="Testing">
      <Results>
        <UnitTestResult>
          <TestDate>2014-09-09T11:17:09</TestDate>
          <Passed>1</Passed>
          <Errors>0</Errors>
          <Failures>0</Failures>
          <Inconclusive>0</Inconclusive>
          <NotRunnable>0</NotRunnable>
          <Skipped>0</Skipped>
          <Ignored>0</Ignored>
          <Time />
          <Message>Test successful

Execution time: 0.04ms</Message>
        </UnitTestResult>
        <UnitTestResult>
          <TestDate>2014-09-09T11:20:56</TestDate>
          <Passed>1</Passed>
          <Errors>0</Errors>
          <Failures>1</Failures>
          <Inconclusive>0</Inconclusive>
          <NotRunnable>0</NotRunnable>
          <Skipped>0</Skipped>
          <Ignored>0</Ignored>
          <Time />
          <Message>One or more child tests had errors</Message>
        </UnitTestResult>
        <UnitTestResult>
          <TestDate>2014-09-09T11:21:12</TestDate>
          <Passed>1</Passed>
          <Errors>0</Errors>
          <Failures>1</Failures>
          <Inconclusive>0</Inconclusive>
          <NotRunnable>0</NotRunnable>
          <Skipped>0</Skipped>
          <Ignored>0</Ignored>
          <Time />
          <Message>One or more child tests had errors</Message>
        </UnitTestResult>
        <UnitTestResult>
          <TestDate>2014-09-09T11:29:07</TestDate>
          <Passed>0</Passed>
          <Errors>0</Errors>
          <Failures>2</Failures>
          <Inconclusive>0</Inconclusive>
          <NotRunnable>0</NotRunnable>
          <Skipped>0</Skipped>
          <Ignored>0</Ignored>
          <Time />
          <Message>One or more child tests had errors</Message>
        </UnitTestResult>
      </Results>
      <Tests>
        <TestRecord Name="Test">
          <Results>
            <UnitTestResult>
              <TestDate>2014-09-09T11:17:09</TestDate>
              <Passed>1</Passed>
              <Errors>0</Errors>
              <Failures>0</Failures>
              <Inconclusive>0</Inconclusive>
              <NotRunnable>0</NotRunnable>
              <Skipped>0</Skipped>
              <Ignored>0</Ignored>
              <Time />
              <Message>Test successful

Execution time: 0.04ms</Message>
            </UnitTestResult>
          </Results>
          <Tests>
            <TestRecord Name="TestCase">
              <Results>
                <UnitTestResult>
                  <TestDate>2014-09-09T11:17:09</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0.02ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
              </Results>
            </TestRecord>
          </Tests>
        </TestRecord>
        <TestRecord Name="MathUtil2DTest">
          <Results>
            <UnitTestResult>
              <TestDate>2014-09-09T11:20:56</TestDate>
              <Passed>1</Passed>
              <Errors>0</Errors>
              <Failures>1</Failures>
              <Inconclusive>0</Inconclusive>
              <NotRunnable>0</NotRunnable>
              <Skipped>0</Skipped>
              <Ignored>0</Ignored>
              <Time />
              <Message>One or more child tests had errors</Message>
            </UnitTestResult>
            <UnitTestResult>
              <TestDate>2014-09-09T11:21:12</TestDate>
              <Passed>1</Passed>
              <Errors>0</Errors>
              <Failures>1</Failures>
              <Inconclusive>0</Inconclusive>
              <NotRunnable>0</NotRunnable>
              <Skipped>0</Skipped>
              <Ignored>0</Ignored>
              <Time />
              <Message>One or more child tests had errors</Message>
            </UnitTestResult>
            <UnitTestResult>
              <TestDate>2014-09-09T11:29:07</TestDate>
              <Passed>0</Passed>
              <Errors>0</Errors>
              <Failures>2</Failures>
              <Inconclusive>0</Inconclusive>
              <NotRunnable>0</NotRunnable>
              <Skipped>0</Skipped>
              <Ignored>0</Ignored>
              <Time />
              <Message>One or more child tests had errors</Message>
            </UnitTestResult>
          </Results>
          <Tests>
            <TestRecord Name="RoundOff_NineToTen">
              <Results>
                <UnitTestResult>
                  <TestDate>2014-09-09T11:20:56</TestDate>
                  <Passed>0</Passed>
                  <Errors>0</Errors>
                  <Failures>1</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>  Expected: 10
  But was:  0.0f
</Message>
                  <StackTrace>   at NUnit.Framework.Assert.That(Object actual, IResolveConstraint expression, String message, Object[] args)
   at Testing.MathUtil2DTest.RoundOff_NineToTen() in c:\Users\Jerome\Google Drive\Projects\Game\NBom\Development\NBomUnity\Testing\GameLib\MathUtil2DTest.cs:line 20</StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2014-09-09T11:21:12</TestDate>
                  <Passed>0</Passed>
                  <Errors>0</Errors>
                  <Failures>1</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>  Expected: 10
  But was:  0.0f
</Message>
                  <StackTrace>   at NUnit.Framework.Assert.That(Object actual, IResolveConstraint expression, String message, Object[] args)
   at Testing.MathUtil2DTest.RoundOff_NineToTen() in c:\Users\Jerome\Google Drive\Projects\Game\NBom\Development\NBomUnity\Testing\GameLib\MathUtil2DTest.cs:line 20</StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2014-09-09T11:29:07</TestDate>
                  <Passed>0</Passed>
                  <Errors>0</Errors>
                  <Failures>1</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>  Expected: 10
  But was:  0.0f
</Message>
                  <StackTrace>   at NUnit.Framework.Assert.That(Object actual, IResolveConstraint expression, String message, Object[] args)
   at Testing.MathUtil2DTest.RoundOff_NineToTen() in c:\Users\Jerome\Google Drive\Projects\Game\NBom\Development\NBomUnity\Testing\GameLib\MathUtil2DTest.cs:line 20</StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2014-09-09T11:29:40</TestDate>
                  <Passed>0</Passed>
                  <Errors>0</Errors>
                  <Failures>1</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>  Expected: 10
  But was:  0.0f
</Message>
                  <StackTrace>   at NUnit.Framework.Assert.That(Object actual, IResolveConstraint expression, String message, Object[] args)
   at Testing.MathUtil2DTest.RoundOff_NineToTen() in c:\Users\Jerome\Google Drive\Projects\Game\NBom\Development\NBomUnity\Testing\GameLib\MathUtil2DTest.cs:line 20</StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2014-09-09T11:30:27</TestDate>
                  <Passed>0</Passed>
                  <Errors>0</Errors>
                  <Failures>1</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>  Expected: 10
  But was:  -10.0f
</Message>
                  <StackTrace>   at NUnit.Framework.Assert.That(Object actual, IResolveConstraint expression, String message, Object[] args)
   at Testing.MathUtil2DTest.RoundOff_NineToTen() in c:\Users\Jerome\Google Drive\Projects\Game\NBom\Development\NBomUnity\Testing\GameLib\MathUtil2DTest.cs:line 20</StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
              </Results>
            </TestRecord>
            <TestRecord Name="RoundOff_OneToZero">
              <Results>
                <UnitTestResult>
                  <TestDate>2014-09-09T11:20:56</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0.00ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2014-09-09T11:21:12</TestDate>
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0.00ms</Message>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2014-09-09T11:29:07</TestDate>
                  <Passed>0</Passed>
                  <Errors>0</Errors>
                  <Failures>1</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>  Expected: 0
  But was:  -10.0f
</Message>
                  <StackTrace>   at NUnit.Framework.Assert.That(Object actual, IResolveConstraint expression, String message, Object[] args)
   at Testing.MathUtil2DTest.RoundOff_OneToZero() in c:\Users\Jerome\Google Drive\Projects\Game\NBom\Development\NBomUnity\Testing\GameLib\MathUtil2DTest.cs:line 14</StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2014-09-09T11:29:27</TestDate>
                  <Passed>0</Passed>
                  <Errors>0</Errors>
                  <Failures>1</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>  Expected: 0
  But was:  -10.0f
</Message>
                  <StackTrace>   at NUnit.Framework.Assert.That(Object actual, IResolveConstraint expression, String message, Object[] args)
   at Testing.MathUtil2DTest.RoundOff_OneToZero() in c:\Users\Jerome\Google Drive\Projects\Game\NBom\Development\NBomUnity\Testing\GameLib\MathUtil2DTest.cs:line 14</StackTrace>
                  <ConsoleOutput />
                  <ConsoleError />
                </UnitTestResult>
              </Results>
            </TestRecord>
          </Tests>
        </TestRecord>
      </Tests>
    </TestRecord>
  </Tests>
</TestRecord>